specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -134,43 +197,138 @@
+    "operationName": {
+      "type": "object",
+      "description": "The operation Name parameter.",
       "properties": {
         "value": {
-          "description": "The ID of the subscriptions that is being renamed",
+          "description": "The operation Name",
           "type": "string",
           "readOnly": true
         }
       }
     },
-    "EnabledSubscriptionId": {
-      "description": "The ID of the subscriptions that is being enabled",
+    "DefaultSupportPlanResponseResult": {
+      "description": "Default Support plan result.",
+      "$ref": "#/definitions/SupportPlan"
+    },
+    "SupportPlanResponseResult": {
+      "description": "Support plan result.",
       "properties": {
         "value": {
-          "description": "The ID of the subscriptions that is being enabled",
+          "description": "The list of Support plan.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/SupportPlan"
+          }
+        },
+        "nextLink": {
+          "description": "The link (url) to the next page of results.",
           "type": "string",
           "readOnly": true
         }
       }
     },
-    "SubscriptionName": {
+    "SupportPlan": {
+      "description": "A SupportPlan resource.",
       "type": "object",
-      "description": "The new name of the subscription.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
       "properties": {
-        "subscriptionName": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SupportPlanProperties",
+          "title": "SupportPlan properties"
+        }
+      }
+    },
+    "SupportPlanProperties": {
+      "description": "The properties of the support plan.",
+      "properties": {
+        "planType": {
+          "$ref": "#/definitions/planTypeEnum"
+        },
+        "endDate": {
           "type": "string",
-          "description": "New subscription name"
+          "description": "the end date"
+        }
+      }
+    },
+    "ProxyResource": {
+      "description": "The Resource model definition.",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource Id."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "OperationStatusResponseResult": {
+      "description": "Purchase Record result.",
+      "properties": {
+        "planType": {
+          "$ref": "#/definitions/planTypeEnum"
+        },
+        "endDate": {
+          "type": "string",
+          "description": "the end date"
+        }
+      }
+    },
+    "SupportPlanName": {
```

Comment:
`SupportPlanProperties`? Since this technically describes an object containing the support plan type.